client/src/components/BoostConfigList.tsx

```diff
@@ -0,0 +1,58 @@
+import { faFile, faTrashAlt } from '@fortawesome/free-solid-svg-icons';
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import { BoostConfig } from 'types/boost';
+import FontAwesomeIcon from './FontAwesomeIcon';
+import WidgetListGroupItem from './WidgetListGroupItem';
+import styles from './BoostConfigList.module.css';
+
+export interface BoostConfigListProps {
+  config: BoostConfig;
+  onSelectConfig: (configName: string) => void;
+  onDeleteConfig: (configName: string) => void;
+}
+
+export default function BoostConfigList({
+  config,
+  onSelectConfig,
+  onDeleteConfig,
+}: BoostConfigListProps) {
+  const handleSelect = (configName: string) => {
+    if (configName !== config.active) onSelectConfig(configName);
+  };
+  const handleDelete = (event: React.MouseEvent, configName: string) => {
+    event.stopPropagation();
+    onDeleteConfig(configName);
+  };
```

Comment:
Should these be `useCallback` because they are used as callbacks/handlers for `onClick`?

Reply:
Good question - this has been something I've been a little confused about in the past. I had a read of this article https://dmitripavlutin.com/dont-overuse-react-usecallback/. From my understanding, using `useCallback` wouldn't be beneficial since we're not actually making any use of memoization. Does that seem right to you?